src/docs/designingComponents.mdx

```diff
@@ -41,3 +41,11 @@ Make sure you aren't duplicating functionality that already exists. Don't hard c
 
 **Call things what they are (even when they are long)**<br/>
 Long names are better than ambiguous names.
+
+## Use Component Template
+
+There is a componet template in the compoents library called CopyPasteThisComponent
```

Comment:
```suggestion
There is a component template in the components library called CopyPasteThisComponent
```

Reply:
See also this comment line on the previous PR:
https://github.com/oaknational/oak-components/blob/main/src/components/CopyPasteThisComponent/CopyPasteThisComponent.stories.tsx#L7